hugegraph-dist/src/assembly/travis/mysql.cnf

```diff
@@ -6,7 +6,7 @@ character_set_server="utf8"
 collation_server="utf8_bin"
 
 # Ensure operations involving astral characters fail loudly,
-# rather than mysql silently replacing each each byte of the
+# rather than mysql silently replacing each byte of the
```

Comment:
This isn't strictly a spelling thing, but spell checkers have been complaining about doubled words for decades, so my tool now includes this as well...

Reply:
nice catch for your spelling-check tool, we do need the rule